stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/header/MessageListHeader.kt

```diff
@@ -86,55 +98,58 @@ public fun MessageListHeader(
             verticalAlignment = Alignment.CenterVertically,
         ) {
 
-            BackButton(
-                modifier = Modifier
-                    .size(36.dp),
-                imageVector = Icons.Default.ArrowBack,
-                onBackPressed = onBackPressed
-            )
+            leadingContent()
 
-            MessagesHeaderTitle(
-                modifier = Modifier
-                    .weight(6f)
-                    .fillMaxHeight()
-                    .clickable(
-                        interactionSource = remember { MutableInteractionSource() },
-                        indication = null,
-                        onClick = onHeaderActionClick
-                    ),
-                title = title,
-                subtitle = subtitle,
-                isNetworkAvailable = isNetworkAvailable
-            )
+            titleContent()
 
-            ChannelAvatar(
-                modifier = Modifier.size(36.dp),
-                channel = channel,
-                currentUser = currentUser,
-                contentDescription = channel.name
-            )
+            trailingContent()
```

Comment:
Love how clean the implementation of this function is now 😄

Reply:
Yeah it's really nice! :]

And we can customize different parts of it to a greater extent which is cool